src/encoded/schemas/matrix_file.json

```diff
@@ -348,7 +390,8 @@
         "txt": ".txt",
         "mtx": ".mtx",
         "rds": ".rds",
-        "mex": ""
+        "mex": "",
+        "hdf5": ".h5ad"
```

Comment:
now we have 2 hdf5 keys here. I made a note about this on LAT-110, would be good to sort out desired outcomes there.

Reply:
Just making sure I understand, I don't need to change this, right?

Reply:
I think you should undo your change. I have a different solution in mind for LAT-110